foca/models/config.py

```diff
@@ -600,11 +620,14 @@ class LogFormatterConfig(FOCABaseConfig):
     """
     class_formatter: str = Field(
         "logging.Formatter",
-        alias="class",
+        env="CLASS",
```

Comment:
Is this to get rid of the warning? Just in case it wasn't clear why `Field()` was used here: we can't call a field `class` because it's a reserved word. However, we _must_ refer to it as `class` or else the log config (as required by the `logging` module) won't work, hence the alias. But when used together with `BaseSettings` (but not `BaseModel`) a warning is thrown, I think because the meaning of `alias` changes somewhat in `BaseSettings`. In any case: I'll need to read up on this behavior and what `env` is used for in `BaseSettings`, but I'm getting a little tired....